contrib/dcr_setup_go_workspace.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
Looks like it needs a `chmod +x contrib/dcr_setup_go_workspace.sh`?

Reply:
Good catch.  Done.